editor/meta-boxes/meta-boxes-iframe/index.js

```diff
@@ -1,265 +0,0 @@
-/**
```

Comment:
The moved `meta-boxes-area/style.scss` file still contains three selectors referencing selectors prefixed with `.editor-meta-boxes-iframe`.